client/api/README.md

```diff
@@ -4,7 +4,7 @@
 
 CSI-proxy's API is a GRPC, versioned API.
 
-The server exposes a number of API groups, all independent of each other. Additionally, each API group has one or more versions. Each version in each group listens for GRPC messages on a Windows named pipe of the form `\\.\\pipe\\csi-proxy-<api_group_name>-<version>` (e.g. `\\.\\pipe\\csi-proxy-filesystem-v2alpha1` or `\\.\\pipe\\csi-proxy-iscsi-v1`).
+The server exposes a number of API groups, all independent of each other. Additionally, each API group has one or more versions. Each version in each group listens for GRPC messages on a Windows named pipe of the form `\\.\\pipe\\csi-proxy-<api_group_name>-<version>` (e.g. `\\.\\pipe\\csi-proxy-file_system-v2alpha1` or `\\.\\pipe\\csi-proxy-iscsi-v1`).
```

Comment:
`FileSystem` seemed cleaner than `Filesystem` in the places where we need camel case, so that's why I changed this - but happy to change back if you prefer.

Reply:
FileSystem seemed cleaner than Filesystem in the places where we need camel case, so that's why I changed this - but happy to change back if you prefer.

Reply:
My preference would be `filesystem`.

Reply:
👍